kidsability/src/containers/Login.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import "../styling/Login.css";
+
+export default function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+
+  // include check for valid email with @ character
+  function validateForm() {
+    var validate = require("react-email-validator")
```

Comment:
put this after the imports and change var to const, I think going forward we should avoid using var altogether and instead use either let or const